### ISSUE TITLE
⌨ Manual Accessibility test of the  `SkillBrowser` component inline version on UAT

### ISSUE THREAD

**Issue by RM-1978** [GCTC-NTGC/gc-digital-talent#8001]
## ⌨ Accessibility Issue
Test `SkillBrowser` component inline version#7352

## 🦋 Expected Behaviour

Component passes manual testing for Narrator, NVDA, VoiceOver. 

## 🕵️ Details

- **Operating Systems**:  Windows, MacOS, iOS, Android. 
- **Browsers**: Edge, Chrome, Safari
- **Assistive Technologies**: Narrator, VoiceOver, TalkBack. 

## ⌚ Timebox 
 - Timebox = 1/2 day

## ✅ Acceptance Criteria

- [ ] No problems are found in any AT Configuration
- [ ] Accessibility issues are created for any found problems

**Comment by esizer** [comment]
Findings,

- Multiselect combobox items set `aria-selected` on the highlighted item, not selected items. This is a bug with downshift and will be fixed in v9. We may need to hack around this
- Chips being below is causes some odd navigation,  this is okay but would be much improved if they were before the input

## Aria state management

Our multi select list should have a `aria-multiselectable=true` on the `ul[role=listbox]`, items in the list should have `aria-selected` values set to `true` when selected and `false` when not selected.

This is likely of high priority since screen readers announce that the item is selected regardless of its actual state since downshift adds `aria-selected=true` to the highlighted item which is very confusing.

**Comment by esizer** [comment]
Tickets drafted in the insight tracker.
